src/app/types/UserType.tsx

```diff
@@ -0,0 +1,20 @@
+export interface User {
```

Comment:
types folder should be in the src folder, not the app folder since the app folder is reserved for the Next app router